### PR TITLE
Add `CheckPaused` rpc

### PR DESCRIPTION
**Pull request by supl**
This PR add a new service `CheckPaused` and a new message `CheckPausedResponse` to get the pause status from the gRPC servers.

I will create other PRs in the scalar and scalardb projects after this PR is merged and scalar-admin is released.

### CODE REVIEW

File: src/main/grpc/com/scalar/admin/rpc/AdminGrpc.java [hunk 1]
```diff
@@ -77,6 +77,37 @@ com.google.protobuf.Empty> getUnpauseMethod() {
     return getUnpauseMethod;
   }
 
+  private static volatile io.grpc.MethodDescriptor<com.google.protobuf.Empty,
```
Comment (supl) [inline src/main/grpc/com/scalar/admin/rpc/AdminGrpc.java:80]: Can ignore this part.
It's generated by the protocol buffers compiler.

File: src/main/java/com/scalar/admin/AdminClient.java [hunk 2]
```diff
@@ -52,6 +52,15 @@ public Optional<String> stats() {
     }
   }
 
+  public Optional<String> paused() {
```
Comment (supl) [inline src/main/java/com/scalar/admin/AdminClient.java:55]: I used Optional<String> as the return type to consistent with the stats() function so the RequestCoordinator can use the same `run` logic.

File: src/main/java/com/scalar/admin/RequestCoordinator.java [hunk 3]
```diff
@@ -52,6 +52,12 @@ public JsonObject stats() {
     return builder.build();
   }
 
+  public JsonObject paused() {
```
Comment (supl) [inline src/main/java/com/scalar/admin/RequestCoordinator.java:55]: The result will be like
```json
{"ip1:port1": "false", "ip2:port2": "false"}
```

File: src/main/proto/scalar/protobuf/admin.proto [hunk 4]
```diff
@@ -13,6 +13,8 @@ service Admin {
     }
     rpc Unpause (google.protobuf.Empty) returns (google.protobuf.Empty) {
     }
+    rpc Paused (google.protobuf.Empty) returns (PausedResponse) {
```
[the remark this answers is not in the document]
Comment (supl) [reply src/main/proto/scalar/protobuf/admin.proto]: Thank you for your advice.
I changed the service name to `checkPaused` in https://github.com/scalar-labs/scalar-admin/pull/10/commits/e6052f9b516dc3f8e65c73c3e7893dfb949b9cbf

The return value is also renamed to be `CheckPausedResponse`.
(it seems that the return value can be a primitive type)

File: build.gradle [hunk 5]
```diff
@@ -88,7 +88,7 @@ targetCompatibility = 1.8
 
 group = "com.scalar-labs"
 archivesBaseName = "scalar-admin"
-version = "1.1.0"
+version = "1.2.0"
```
[the remark this answers is not in the document]
Comment (supl) [reply build.gradle]: Changed it back in https://github.com/scalar-labs/scalar-admin/pull/10/commits/93a6b8599f6783e9f8149fcf842fdc86a31590e6

**Review by brfrn169 — APPROVED** [review]
LGTM! Thank you!

File: src/main/java/com/scalar/admin/Command.java [hunk 6]
```diff
@@ -3,5 +3,6 @@
 public enum Command {
   PAUSE,
   UNPAUSE,
-  STATS
+  STATS,
+  CHECKPAUSED
```
Comment (feeblefakie) [inline src/main/java/com/scalar/admin/Command.java:7]: Sorry, it's a bit picky, but it should be `CHECK_PAUSED`.

**Review by feeblefakie — APPROVED** [review]
LGTM! Left one suggestion, so please merge after applying it.

File: src/main/java/com/scalar/admin/Command.java (hunk 6 above)
Comment (supl) [reply src/main/java/com/scalar/admin/Command.java]: Sorry, my bad.
It's renamed in https://github.com/scalar-labs/scalar-admin/pull/10/commits/a557b584d7348230b228021f5b911d384b6e7570

**Pull request merged**